api/v2/sdks/sdk-ref-csharp-dotnet.md

```diff
@@ -45,53 +51,103 @@ The binary is published in this location:
 
 ## Initialization
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the EUID service. It also allows you to configure retry intervals in the event of errors.
+DSPs should create an instance of the `BidstreamClient` class.
+
+You will need to provide the values necessary for the SDK to authenticate with the EUID service.
+
+| Parameter | Description |
+| :--- | :--- |
+| `endpoint` | The endpoint for the EUID service. See [Environments](../getting-started/gs-environments). | 
+| `authKey` | The API key. See [EUID Credentials](../getting-started/gs-credentials). |
+| `secretKey` | The client secret. See [EUID Credentials](../getting-started/gs-credentials). |
+
+## Interface
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the EUID service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to EUID, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+The `BidstreamClient` class allows you to decrypt EUID tokens into raw EUIDs.
 
-## Interface 
+For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
 
-The interface allows you to decrypt EUID advertising tokens and return the corresponding raw EUID. 
+The `SharingClient` class allows you to encrypt raw EUIDs into EUID tokens and decrypt EUID tokens into raw EUIDs.
```

Comment:
Shall we modify it like:
_The `SharingClient` class is not available for use in EUID and therefore can be ignored for now_

So we don't even mention about the functionalities not supported by EUID but at least clearly explicitly tell EUID participants to ignore it?

Reply:
@sunnywu I have a question on that. Basically, it depends whether it's visible to EUID people? AFAIK it's the same SDK and therefore, yes, if there is functionality there that isn't in use for EUID we should note it. Is that the case here?

Reply:
Yes your assumption and result looks right!